product_security/recommendations_for_customers/secure_container_lifecycle_management.adoc

```diff
@@ -0,0 +1,96 @@
+[[dedicated-secure-container-lifecycle-management]]
+=  Secure Container Lifecycle Management
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+Securing containers is a lot like securing any running process. As an
+application owner, you must think about security throughout the layers of the
+solution stack before you deploy and run your container. You also need to think
+about security throughout the application and container lifecycle. The
+{product-title} team provides secure platform management and tools to help and
+securely deploy and manage containers. This topic discusses key elements of
+container lifecycle management that you must think about.
+
+[[container-host-multi-tenancy]]
+== Container Host Multi-tenancy
+
+Containers are Linux® processes with isolation and resource confinement that
+enable you to run sandboxed applications on a shared host kernel.
+{product-title} makes use of the multiple levels of security available in Red
+Hat Enterprise Linux Host, which is the underlying immutable operating system
+deployed with {product-title}. Linux features such as Security-Enhanced Linux,
+namespaces, Cgroups, and seccomp are used to secure the container. Containers
+are run with user, without a fixed UID, and not root privileges on the host.
```

Comment:
Replace with - Container processes run under a UID assigned by the OpenShift Platform, and never with root privileges.